rom/laghos_utils.cpp

```diff
@@ -40,15 +40,16 @@ int WriteOfflineParam(int dim, double dt, ROM_Options& romOptions,
         VerifyOfflineParam(dim, dt, romOptions, numWindows, twfile, paramfile, true);
     }
 
-    std::ofstream opout(paramfile, std::fstream::app);
     if (printStatus)
     {
+        std::ofstream opout(paramfile, std::fstream::app);
         opout << romOptions.parameterID << " ";
         opout << romOptions.rhoFactor << " ";
         opout << romOptions.blast_energyFactor << " ";
         opout << romOptions.atwoodFactor << endl;
         opout.close();
     }
+    return 0;
```

Comment:
What is the purpose the return value? It seems this function, as well as `VerifyOfflineParam` could be `void`.

Reply:
Yes it is meaningless and should be removed.